### PR TITLE
gives service borgs back their tray

### PR DESCRIPTION
**Pull request by Ohagi-Chan**
## About The Pull Request
Reverts part of https://github.com/austation/austation/pull/2411
The tray is one of my favourite tools as a cook and I don't see why it was removed from the service borg.

## Why It's Good For The Game
Trays are really nice to use, gives serviceborg back a powerful tool

## Changelog
:cl:
add: service borgs can use the tray module again
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
You should read the part of the PR you're reverting xd. The tray was replaced with the beaker apparatus which can do everything that the tray can do

**Comment by ZombiesVsPlants** [comment]
is it too much to ask for both?

**Comment by Ohagi-Chan** [comment]
> You should read the part of the PR you're reverting xd. The tray was replaced with the beaker apparatus which can do (almost) everything that the tray can do

I disagree.
The tray can hold up to 7 small items at a time and has the same quick-move qol features as backpacks.
Trays are the best way to move a box of donk pockets out of the box and into a microwave, in less than a second.

Trays are great in a pinch for moving groups of objects around in botany and generally awesome for setting tables.

File: austation/code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -260,6 +260,10 @@
 			return FALSE
 	return ..()
 
+/obj/item/robot_module/butler/Initialize()
+	..()
+	basic_modules |= /obj/item/storage/bag/tray
```
Comment (MCterra10) [inline austation/code/modules/mob/living/silicon/robot/robot_modules.dm:265]: oh god yes
look at this modularization
image

**Review by MCterra10 — APPROVED** [review]

File: austation/code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (Ohagi-Chan) [reply austation/code/modules/mob/living/silicon/robot/robot_modules.dm:265]: I don't get it, sorry.

Comment (MCterra10) [reply austation/code/modules/mob/living/silicon/robot/robot_modules.dm:265]: good modularization makes me wanna coooooom

Comment (Ohagi-Chan) [reply austation/code/modules/mob/living/silicon/robot/robot_modules.dm:265]: Reading this again, should I instead make it +=, rather than |= ?

Comment (Ohagi-Chan) [reply austation/code/modules/mob/living/silicon/robot/robot_modules.dm:265]: Or maybe keep the |= operator and make it a list containing 1 item?

Comment (MCterra10) [reply austation/code/modules/mob/living/silicon/robot/robot_modules.dm:265]: |= is the correct way to do this

**Pull request merged**